pom.xml

```diff
@@ -264,6 +261,26 @@
           <compilerArgument>${compilerArgument}</compilerArgument>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M1</version>
```

Comment:
Why are we using an unstable version? Is this an unstable version?

Reply:
The [Maven Site](https://maven.apache.org/enforcer/download.cgi) says that _Apache Maven Enforcer 3.0.0-M1_ is the current stable version of Apache Maven Enforcer.